.github/workflows/publish-all-platforms.yml

```diff
@@ -125,6 +125,7 @@ jobs:
   publish-golang:
     needs: 
       - setup
+      - build-android
```

Comment:
In order to build android don't we need also to add also the golang condition in the setup.android output variable?

Reply:
Like this?
```
      android: ${{ !!inputs.kotlin-mpp-package-version || !!inputs.maven-package-version || !!inputs.golang-package-version }}
```

Reply:
Ah.. I see now you don't need it because you copy the binaries directly.
In the flutter publish I am copying the jniLibs which is built on android workflow that includes the c++ shared.

Reply:
I think you're right @roeierez. As publish-golang downloads the artifacts uploaded by the build job of build-android.  Can you double check @JssDWt?

Reply:
Yes! Correct, I missed that. The artifacts are only available for the workflow you're running, so if golang needs android, it needs android to build.